origins_unique_title/origins_unique_title.module

```diff
@@ -26,7 +27,7 @@ function origins_unique_title_form_node_form_alter(&$form, FormStateInterface $f
     '#attributes' => [
       'id' => 'title-unique'
     ],
-    '#weight' => $form['title']['#weight'] + 1,
+    '#weight' => $form['title']['#weight'] + 0.5,
```

Comment:
I know Drupal's Form API pushes heavier weighted items down the render order. Is there a tangible difference of behaviour between 1 and 0.5?

Reply:
Partly flagging this as a non-standard approach but also curiousity. If it's benign/not needed suggest we omit in this PR

Reply:
+1 was setting the error message below the field that comes after the title. This happens on NIDirect also.
Setting it to +0.5 set it after the title field where is should be.
I was going to set it = 0 but I wasn't sure if there would be any cases where the title field wasn't the 1st field in the form?
Th